README.md

```diff
@@ -1,12 +1,20 @@
 Bulk User Management
 ====================
 
-A WordPress plugin that lets you manage users across all your sites from one place on a multisite install.
+Bulk User Management allows a set of specified users to manage permissions across all your sites. Users from every site in your network are displayed in a list and can be added to or removed from any of your sites. Any users that aren’t currently in your network can still be invited to any of your blogs through the normal invite process. After they accept, they can be managed in bulk across the network.
+
+To enable the plugin, you will need to pass an array of user logins to `wpcom_vip_bulk_user_management_whitelist()`. Bulk User Management will only be available for these users and they will be able to manage all the users in your network regardless of their specified capability on those sites.
+
+```
+wpcom_vip_bulk_user_management_whitelist( array( 'user1', 'user2', 'user3' ) );
+```
```

Comment:
I'd suggest moving this content to a new `WordPress VIP` section under the Filters section. 

The content here can be replaced with text that is suitable for anyone running the plugin anywhere. i.e. "To enable the plugin, you will need to pass an array of sites, user IDs or usernames using the filters below. Bulk User Management will only be ..."

The WordPress VIP section can then be framed along the lines of "If you're running this plugin on https://wpvip.com, then you can use the convenience function `wpcom_vip_bulk_user_management_whitelist()` with an array of user IDs:" followed by the code snippet.